src/main/java/race/Car.java

```diff
@@ -6,14 +6,44 @@
 
 public class Car {
 
-    private List<Boolean> distance = new ArrayList<>();
 
-    public List<Boolean> getDistance() {
-        return Collections.unmodifiableList(distance);
+    private String name;
+
+    private List<Boolean> winOrLoseLog = new ArrayList<>();
+
+    private int distance;
+
```

Comment:
```suggestion
    private String name;
    private List<Boolean> winOrLoseLog = new ArrayList<>();
    private int distance;

```
멤버변수들은 붙여서 사용해주면 어떨까요?
그리고 distnace는 winOrLose로 대체할 수 있지 않을까 싶어요!